### DISCUSSION TITLE
need tflite ep for android

### DISCUSSION THREAD

**Discussion by WangHHY19931001** [microsoft/onnxruntime#23034] [Ideas / Feature Requests]
https://developer.android.google.cn/ndk/guides/neuralnetworks/migration-guide

nnapi will gone, so we need tflite ep

**Comment by RMichaelPickering** [comment]
tflite is now renamed as LiteRT, but with nnapi deprecated this request should be prioritized! Or please update the roadmap to clarify how good EP support will be provided for Android devices.
